cmd/stack_select.go

```diff
@@ -86,5 +97,8 @@ func newStackSelectCmd() *cobra.Command {
 	cmd.PersistentFlags().StringVarP(
 		&stack, "stack", "s", "",
 		"The name of the stack to select")
+	cmd.PersistentFlags().BoolVar(
```

Comment:
```suggestion
	cmd.PersistentFlags().BoolVarP(
		&create, "create", "c", false,
		"If selected stack does not exist, create it")
```

Reply:
I was worried about confusion between global flag -C and -c, what do you think?

Reply:
I wouldn't worry about it. Case differences in short-form flags that have different effects are pretty common.

Reply:
Thank you!